### ISSUE TITLE
Fix A11y issues

### ISSUE THREAD

**Issue by mewdriller** [b2io/base2.io#323]
- [ ] Fix errors reported in [`pa11y`](https://github.com/pa11y/pa11y) audit; `pa11y http://base2.io`
- [ ] Fix errors reported in [`lighthouse`](https://github.com/GoogleChrome/lighthouse) audit; `lighthouse http://base2.io --view`

[`lighthouse` Audit Results](https://github.com/b2io/base2.io/files/2397774/www.base2.io_2018-09-19_11-46-27.report.pdf)

<details>
<summary><code>pa11y</code> Audit Results</summary>

```

Welcome to Pa11y

 > Running Pa11y on URL http://base2.io

Results for URL: http://www.base2.io/

 • Error: This button element does not have a name available to an accessibility API. Valid names are: title attribute, element content, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name
   ├── #___gatsby > main > nav > div:nth-child(1) > div > button
   └── <button class="sc-Rmtcm jaTawv sc-cSHVUG fiuFXh"><svg fill="currentColor" preser...</button>

 • Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #___gatsby > main > header > a
   └── <a class="sc-gqjmRU gJOwwz" href="/#next" title="Scroll down for more"><img class="sc-bYSBpT gjoRRK" s...</a>

 • Error: Duplicate id attribute value "path0_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path0_fill
   └── <path id="path0_fill" fill-rule="evenodd" d="M21.416 29.662c2.19-2.573 3.285-6.06 3.285-10.461v-4.879c0-4.386-1.102-7.87-3.307-10.45C19.19 1.29 16.167 0 12.328 0 8.473 0 5.455 1.29 3.273 3.872 1.09 6.452 0 9.936 0 14.322v4.879c0 4.401 1.098 7.888 3.2...

 • Error: Duplicate id attribute value "path1_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path1_fill
   └── <path id="path1_fill" d="M8.084 24.92l1.418 5.28h.134l1.418-5.28L18.89 0h.921l8.025 24.92 1.395 5.28h.135l1.462-5.28L37.78 0h1.382l-9.076 32.528h-1.372L20.593 5.361l-1.08-3.245h-.135L18.343 5.36l-8.189 27.167H8.782L0 0h1.29l6.794 24.92z"></path>

 • Error: Duplicate id attribute value "path2_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path2_fill
   └── <path id="path2_fill" d="M24.807 1.336H13.28v31.165h-1.382V1.336H0V0h24.807v1.336z"></path>

 • Error: Duplicate id attribute value "path3_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path3_fill
   └── <path id="path3_fill" d="M17.047 15.932H1.382v15.233H19.35v1.267H0V0h19.35v1.267H1.383v13.399h15.665v1.266z"></path>

 • Error: Duplicate id attribute value "path4_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path4_fill
   └── <path id="path4_fill" d="M21.972 25.377c0-1.864-.77-3.368-2.306-4.51-1.537-1.141-4.159-2.316-7.863-3.524-3.689-1.119-6.407-2.305-8.155-3.558-1.747-1.253-2.62-3.11-2.62-5.572 0-2.447 1.023-4.428 3.07-5.942C6.145.757 8.744 0 11.894 0c3.269 0 5.946.78 7...

 • Error: Duplicate id attribute value "path5_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path5_fill
   └── <path id="path5_fill" fill-rule="evenodd" d="M4.994 22.915H20.99l3.667 9.613h1.327L13.723 0h-1.485L0 32.528h1.327l3.667-9.613zM20.54 21.54H5.444l6.862-18.193.607-1.903h.135l.63 1.903 6.861 18.193z"></path>

 • Error: Duplicate id attribute value "path6_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path6_fill
   └── <path id="path6_fill" fill-rule="evenodd" d="M0 32.54V0h8.754c3.389 0 6.12.65 7.996 2.038C18.624 3.426 19.81 5.5 19.81 8.25c0 1.833-.695 3.426-1.588 4.709-.892 1.283-2.156 2.14-3.79 2.573l-.023.135c1.935.253 3.562 1.137 4.882 2.652 1.32 1.514 1.98 3....

 • Error: Duplicate id attribute value "path7_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path7_fill
   └── <path id="path7_fill" d="M.227 8.821a.225.225 0 0 1-.16-.386L8.48.067a.228.228 0 0 1 .322 0 .225.225 0 0 1 0 .32L.388 8.756a.226.226 0 0 1-.16.065z"></path>

 • Error: Duplicate id attribute value "path8_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path8_fill
   └── <path id="path8_fill" d="M.227 8.822a.226.226 0 0 1-.16-.386L8.48.066a.227.227 0 0 1 .322 0 .225.225 0 0 1 0 .32L.388 8.756a.228.228 0 0 1-.16.066z"></path>

 • Error: Duplicate id attribute value "path9_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path9_fill
   └── <path id="path9_fill" d="M.227 14.4a.225.225 0 0 1-.16-.386L14.089.066a.228.228 0 0 1 .321 0 .225.225 0 0 1 0 .32L.388 14.334a.228.228 0 0 1-.16.066z"></path>

 • Error: Duplicate id attribute value "path10_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path10_fill
   └── <path id="path10_fill" d="M.227 14.401a.226.226 0 0 1-.16-.386L14.088.066a.228.228 0 0 1 .32 0 .225.225 0 0 1 0 .32L.388 14.335a.225.225 0 0 1-.16.066z"></path>

 • Error: Duplicate id attribute value "path11_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path11_fill
   └── <path id="path11_fill" d="M.227 18.12a.226.226 0 0 1-.16-.386L17.827.066a.228.228 0 0 1 .322 0 .225.225 0 0 1 0 .32L.388 18.054a.227.227 0 0 1-.16.066z"></path>

 • Error: Duplicate id attribute value "path12_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path12_fill
   └── <path id="path12_fill" d="M.227 18.12a.226.226 0 0 1-.16-.385L17.828.066a.228.228 0 0 1 .322 0 .226.226 0 0 1 0 .32L.387 18.054a.227.227 0 0 1-.16.067z"></path>

 • Error: Duplicate id attribute value "path13_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path13_fill
   └── <path id="path13_fill" d="M.228 20.91a.227.227 0 0 1-.161-.066.225.225 0 0 1 0-.32L20.632.066a.227.227 0 0 1 .32.32L.389 20.844a.229.229 0 0 1-.16.066z"></path>

 • Error: Duplicate id attribute value "path14_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path14_fill
   └── <path id="path14_fill" d="M.227 22.77a.225.225 0 0 1-.16-.067.224.224 0 0 1 0-.32L22.502.067a.228.228 0 0 1 .321 0 .225.225 0 0 1 0 .32L.388 22.703a.227.227 0 0 1-.16.067z"></path>

 • Error: Duplicate id attribute value "path15_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path15_fill
   └── <path id="path15_fill" d="M.228 23.7a.227.227 0 0 1-.161-.386L23.437.066a.228.228 0 0 1 .32 0 .225.225 0 0 1 0 .32L.389 23.633a.227.227 0 0 1-.16.066z"></path>

 • Error: Duplicate id attribute value "path16_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path16_fill
   └── <path id="path16_fill" d="M.227 22.77a.227.227 0 0 1-.16-.386L22.5.066a.228.228 0 0 1 .322 0c.09.089.09.232 0 .32L.387 22.703a.227.227 0 0 1-.16.066z"></path>

 • Error: Duplicate id attribute value "path17_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path17_fill
   └── <path id="path17_fill" d="M.227 23.7a.225.225 0 0 1-.16-.386L23.437.066a.228.228 0 0 1 .322 0 .225.225 0 0 1 0 .32L.388 23.633a.228.228 0 0 1-.16.067z"></path>

 • Error: Duplicate id attribute value "path18_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path18_fill
   └── <path id="path18_fill" d="M.227 23.7a.225.225 0 0 1-.16-.386L23.437.066a.226.226 0 1 1 .322.32L.388 23.633a.228.228 0 0 1-.16.067z"></path>

 • Error: Duplicate id attribute value "path19_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path19_fill
   └── <path id="path19_fill" d="M.227 20.91a.226.226 0 0 1-.16-.386L20.632.066a.229.229 0 0 1 .321 0c.09.088.09.231 0 .32L.388 20.843a.227.227 0 0 1-.16.067z"></path>

 • Error: Duplicate id attribute value "path20_fill" found on the web page.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_1.F77
   ├── #path20_fill
   └── <path id="path20_fill" d="M33.17 16.499c0 9.112-7.423 16.5-16.584 16.5C7.426 32.998 0 25.61 0 16.498S7.426 0 16.586 0C25.747 0 33.17 7.387 33.17 16.499z"></path>

 • Error: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.
   ├── WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
   ├── #___gatsby > main > section:nth-child(8) > div:nth-child(1) > a:nth-child(6)
   └── <a class="sc-gqjmRU gJOwwz" href="/#contact-us" title="Scroll down for more"><img class="sc-hEsumM fFcjYp" s...</a>

 • Error: This text input element does not have a name available to an accessibility API. Valid names are: label element, title attribute, aria-label attribute, aria-labelledby attribute.
   ├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
   ├── #contact-us > div > form > input:nth-child(1)
   └── <input type="text" name="_gotcha" style="display:none">

 • Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
   ├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
   ├── #contact-us > div > form > input:nth-child(1)
   └── <input type="text" name="_gotcha" style="display:none">

26 Errors
```
</details>
